persistence/src/test/java/edu/unc/lib/dl/persist/services/move/MoveObjectsServiceTest.java

```diff
@@ -250,6 +253,54 @@ public void testMoveObject() throws Exception {
         verifyLogMessage(sourcePid, movePids);
     }
 
+    @Test
+    public void testMoveCollectionIntoWork() {
+        expectedException.expect(IllegalArgumentException.class);
+        expectedException.expectMessage("is not a file and cannot be added to a work");
+
+        WorkObject mockWorkObj = mock(WorkObject.class);
+        when(repositoryObjectLoader.getRepositoryObject(destPid)).thenReturn(mockWorkObj);
+
+        PID movePid = makePid();
+        CollectionObject moveObj = mock(CollectionObject.class);
+        when(repositoryObjectLoader.getRepositoryObject(movePid)).thenReturn(moveObj);
+
+        List<PID> movePids = asList(movePid);
+        service.moveObjects(mockAgent, destPid, movePids);
+    }
+
+    @Test
+    public void testMoveAdminUnitIntoFolder() {
+        expectedException.expect(IllegalArgumentException.class);
+        expectedException.expectMessage("is not a folder or a work and cannot be added to a folder");
+
+        FolderObject mockWorkObj = mock(FolderObject.class);
```

Comment:
very minor, but the variable name is indicating the wrong object type.

Reply:
ha, good catch!